src/test/java/com/github/hey_world_team/currency_converter/repository/CurrencyDataRepositoryTest.java

```diff
@@ -0,0 +1,31 @@
+package com.github.hey_world_team.currency_converter.repository;
+
+import static org.junit.Assert.*;
+
+public class CurrencyDataRepositoryTest {
```

Comment:
It's just an empty class, why is it in the main branch

Reply:
Done!